lib/test/webpack-config.js

```diff
@@ -34,6 +34,11 @@ module.exports = function getStripesWebpackConfig(stripeCore, stripesConfig, opt
     );
   }
 
+  // Remove HMR plugin during testing
+  config.webpack.plugins = config.webpack.plugins.filter(plugin => {
```

Comment:
`config` here is the `webpack` object. So this should be `config.plugins`.

Reply:
Thanks; fixed